sci-libs/fftw/fftw-3.3.8.recipe

```diff
@@ -151,17 +158,15 @@ fi
 
 INSTALL()
 {
-	if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
-		for f in double long-double quad float; do
-	else
-		for f in double long-double float; do
-	fi
-			cd $f
-			make install
-			cd ..
-		done
-
-	rm $libDir/libfftw*.la
+	DIRS="double long-double float"
+ 	if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
+		DIRS="$DIRS quad"
+ 	fi
```

Comment:
Thanks!

Reply:
I think part of this could also be used in TEST()?

Reply:
I guess you are right. This would be nice btw.
So, if using `DIRS` in `TEST()` is desirable than you can move the definition of `DIRS` (and `fftwLibs`) above `INSTALL()`:

```diff

DIRS="double long-double float"
fftwLibs="libfftw3 libfftw3_threads libfftw3f libfftw3f_threads libfftw3l libfftw3l_threads"
if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
	DIRS="$DIRS quad"
	fftwLibs="$fftwLibs libfftw3q libfftw3q_threads"
fi

INSTALL()
{
	prepareInstalledDevelLibs $fftwLibs
 	fixPkgconfig
```

Reply:
That would be even better :)